gradle/versions.gradle

```
@@ -13,8 +13,9 @@
 version = '0.22.0-SNAPSHOT'
 
 ext.versions = [
-	'xtend_lib': '2.28.0',
-	'guava': '[30.1,31)',
+	'xtend_lib': '2.32.0.M2',
+
+	'guava': '[32.1.1,33)',

```

For some reason suggestion is defaulting to adding a newline at the end - this suggestion removes the one I accidentally added earlier.

```suggestion
	'xtend_lib': '2.32.0.M2',
	'guava': '[32.1.1,33)',
```